README.md

```diff
@@ -93,6 +93,31 @@ defmodule YourApp.Router do
     ...
 ```
 
+## Manually notify of exception
+
+If your controller action manually handles an error, the notifier will never be run.
+
+To manually notify of an error you can use `BoomNotifier` in your controller and call the `manual_notify_error/2` callback:
+
+```elixir
+defmodule MyController do
+  use BoomNotifier
```

Comment:
I feel this should be an `import` instead of a `use`. So, you can avoid all the extra complexity of checking if we are in the context of a router or a controller in `BoomNotifier.__using__`.

In fact, it is probably fine to say that the user should invoke `BoomNotifier.notify_error` directly (the user can or not add an `import`)

Reply:
Well, the only thing to solve here is to make sure that [config validations](https://github.com/wyeworks/boom/pull/76/files#diff-e2c95e236410f1e5de0b16a60466a8265ac18f2a015ab4585e0a3495dc9db9ceR70) runs at least once. If the module is used in the Router is fine, but we should make it mandatory (even if it is not going to be used there).

Another option would be to check configs directly in the new `Config` module, as part of any of the functions defined there.. or maybe as an operation that is done only once and cached.